.all-contributorsrc

```diff
@@ -667,7 +667,6 @@
       "avatar_url": "https://avatars3.githubusercontent.com/u/1058963?v=4",
       "profile": "https://thingsthat.com",
       "contributions": [
-        "docs",
```

Comment:
This fails with unknown type in the generation process.

Reply:
Nice catch, thank you!